data/tooltips.txt

```diff
@@ -155,6 +155,9 @@ tip "jump speed:"
 tip "jump fuel:"
 	`How much fuel is consumed when you jump between systems using this drive.`
 
+tip "maintenance cost:"
+	`How many credits per day you need to spend to maintain this outfit.`
```

Comment:
"maintenance costs" suggests costs that you pay for just owning something, regardless of if it is in use. The implementation here only looks at non-parked ships, so I think the implementation below better fits the description "operating costs".

I would like to have both btw, maintenance costs for just owning something, and operating costs for having something active on a ship. (And they can show up on a single item in the accounts overview, no need to separate them there.)

Reply:
>I would like to have both btw

That shouldn't be a difficult change.